docs/BreakingChanges/049 ResolveAssemblyReference task now warns on dependencies with the wrong architecture.md

```diff
@@ -0,0 +1,26 @@
+## 49: ResolveAssemblyReference task now warns on dependencies with the wrong architecture
+
+### Scope
+Minor
+
+### Version Introduced
+4.5.1
+
+### Change Description
+The task emits a warning, MSB3270, which indicates that a reference or any of its dependencies does not match the app's architecture. For example, this occurs if an app that was compiled with the anycpu option includes an x86 reference. Such a scenario could result in an app failure at run time (in this case, if the app is deployed as an x64 process). 
+
+- [ ] Quirked
+- [x] Build-time break
+- [ ] Source analyzer available
+- [x] Source analyzer planned
```

Comment:
Ah, this is new. Some of the breaking change files in the previous PR (#30-39) might be able to use this new field.